app/src/main/java/com/cornellappdev/android/eatery/LoginFragment.java

```diff
@@ -0,0 +1,222 @@
+                    BrbInfoModel model = JsonUtilities.parseBrbInfo(accountInfo);
+                    mAccountPresenter.setBrbModel(model);
+                    mAccountPresenter.setLoggingIn(false);
+                    // If user is still viewing this fragment
+                    if (getFragmentManager() != null) {
+                        loadAccountPage(false);
+                    }
+                }
+            };
+            // Basically, the way we get credentials is to load a specific url on a webview and
+            // execute javascript to submit form data with the user's netid and password. Then,
+            // we retrieve a session_id from the resulting url
+            MainActivity.sLoginWebView.getSettings().setJavaScriptEnabled(true);
+            MainActivity.sLoginWebView.setWebViewClient(new WebViewClient() {
+
+                @Override
+                public void onPageFinished(WebView view, String url) {
+                    // Gets called on every page redirect - initial redirect is loadurl(...)
+                    GetLoginUtilities.loginBrb(url, view, callback);
+                }
+            });
+            mLoginButton.setOnClickListener(new View.OnClickListener() {
+                public void onClick(View v) {
+                    mAccountPresenter.setLoggingIn(true);
+                    loadingGUI();
+                    mAccountPresenter.setNetID(mNetID.getText().toString());
+                    mAccountPresenter.setPassword(mPassword.getText().toString());
+                    // set the context to the context on
+                    // which the button was clicked. This context will then be used for file
+                    // writing later
+                    mAccountPresenter.setContext(getContext());
+                    // change the login javascript to have the correct username and password
+                    mAccountPresenter.resetLoginJS();
+
+                    MainActivity.sLoginWebView.loadUrl(getString(R.string.getlogin_url));
+                }
+            });
+        }
+        return rootView;
+    }
+
+    /**
+     * This method is called simply to display that the GUI is loading. When the user tries to log
+     * in with get, loadingGUI is called. This method disables all textInputs and displays the
+     * loading progress bar
+     */
+    private void loadingGUI() {
+        mProgressBar.setVisibility(View.VISIBLE);
+        mLoginButton.setBackgroundColor(getResources().getColor(R.color.fadedblue));
+        mLoginButton.setText("");
+        mDescriptionText.setTextColor(getResources().getColor(R.color.closed));
+        mDescriptionText.setText("Logging in. This may take a minute ...\n");
+        mNetID.setEnabled(false);
+        mPassword.setEnabled(false);
+        mSaveInfoCheck.setEnabled(false);
+        mPrivacy.setEnabled(false);
+    }
+
+    /*
+     *   Called to make sure the UI of this page is all in the correct display for the user to
```

Comment:
should only be one space between `*` and the start of the comment, so `* Called...`